docs/getting-started-with-scalardb.md

```diff
@@ -1,24 +1,173 @@
 # Getting Started with ScalarDB
 
-Here we assume Oracle JDK 8 and the underlying storage/database such as Cassandra are properly configured.
-If you haven't done it, please configure them first by following [this](getting-started.md).
+This getting started tutorial explains how to configure your preferred database in ScalarDB and set up a basic electronic money application.
+
+{% capture notice--warning %}
+**Warning**
+
+This electronic money application is simplified for this tutorial and isn't suitable for a production environment.
+{% endcapture %}
+
+<div class="notice--warning">{{ notice--warning | markdownify }}</div>
```

Comment:
This is a notice/warning for users. I've added other notices to this document as well, which follow the same syntax.

To see what a notice looks like, please see PR https://github.com/scalar-labs/scalardb/pull/990#issue-1838571367.